tests/unit/amo/pages/TestGuides.js

```diff
@@ -2,17 +2,21 @@ import { oneLine } from 'common-tags';
 import * as React from 'react';
 
 import NotFound from 'amo/components/ErrorPage/NotFound';
+import GuidesAddonCard from 'amo/components/GuidesAddonCard';
 import HeadLinks from 'amo/components/HeadLinks';
 import { fetchGuidesAddons } from 'amo/reducers/guides';
+import Guides, { extractId, GuidesBase, getContent } from 'amo/pages/Guides';
 import {
   dispatchClientMetadata,
+  fakeAddon,
   fakeI18n,
   shallowUntilTarget,
 } from 'tests/unit/helpers';
-import Guides, { extractId, GuidesBase, getContent } from 'amo/pages/Guides';
 
 describe(__filename, () => {
   const getProps = ({
+    addon = fakeAddon,
+    addons = { '{446900e4-71c2-419f-a6a7-df9c091e268b}': addon },
```

Comment:
As a rule of thumb, you should never pass state-mapped props to a component under test. You should dispatch an action to load the data instead.

Try to rewrite these tests with that in mind. You will need to pass a slug param (like `privacy`) then dispatch `loadAddonResults` just like [your saga](https://github.com/mozilla/addons-frontend/blob/d42054537994709bca81eaec6a5344a51f739ab7/src/amo/sagas/guides.js#L30) does. This will test your component in a way similar to how it will be used in real life. Also, this is why your new approach will make testing `GuidesAddonCard` a lot simpler, i.e. you won't have to dispatch loading actions when testing that component.

Reply:
yes, this was exactly my problem; I wasn't sure how to get the var that was set in `mapStateToProps`. I *thought* I tried something similar before but maybe I skipped a step :/. Anyway, I revisited this again and I got it working following your steps (yay!).